lib/players.rb

```diff
@@ -0,0 +1,29 @@
+class Players
+  attr_accessor :name, :piece
+  def initialize
+    @name = name
+    @piece = piece  
+  end
+ 
+  def start_game

```

Comment:
## REQUIRED

Move out this method to `#bin/main.rb` file.

```ruby
#bin/main.rb


puts 'Welcome to TIC-TAC-TOE'
loop do
  puts 'Input name: '
  @name = gets.chomp.capitalize
  puts 'choose x or o'
  @piece = gets.chomp.downcase

  value = %w[x o]
  if @piece == value[0]
    puts "#{@name} is #{@piece}"
  elsif @piece == value[1]
    puts "#{@name} is #{@piece}"
    puts 'Lets play'
    break
  else
    puts 'Invalid choice, choose X OR O!'
  end
end
```
## Expected Output behaviour

```ruby
$ ruby main.rb
Welcome to TIC-TAC-TOE
Input name: 
Bola
choose x or o
X
Bola is x
Input name: 
Ahmed
choose x or o
O
Ahmed is o
Lets play
```